packages/nameguard-react/src/components/Share/Share.tsx

```diff
@@ -137,12 +136,10 @@ export function Share({ name }: ShareProps) {
 
                   <div className="min-h-[200px] flex items-center justify-center px-6 md:px-10">
                     {name && (
```

Comment:
Why are we checking here if `name` is defined? It should always be defined based on the defined props of this UI component.